Makefile.am

```diff
@@ -48,6 +48,8 @@ endif
 
 EXTRA_DIST += autogen.sh COPYING README.md
 
+DEFS += -DTARGET_PREFIX="\"$(prefix)\""
```

Comment:
Huh, I'd never heard of `DEFS` before.  Let's instead add to `AM_CPPFLAGS`, since we use that above this for very similar things like `-DDATADIR='"$(datadir)"'`.